### DISCUSSION TITLE
Could you remove overflow: hidden on https://statecharts.io/inspect

### DISCUSSION THREAD

**Discussion by Dema** [statelyai/xstate#2732] [General] [answered]
I have a pretty big context (like 300 lines of json) and inspector doesn't allow me to scroll to the bottom of it. I have to manually switch off `overflow: hidden` in `[data-xviz=service-sidebar]>[data-reach-tabs] [data-reach-tab-panels]` every time it refreshes. Could you make is scrollable?

**Comment by Andarist** [comment, marked as answer]
This is the old inspector URL and it's unlikely that we are going to change anything there. Please use https://stately.ai/viz?inspect=1 for the time being. We are going to update the default URL soon-ish
